docs/data-management.rst

```diff
@@ -154,50 +202,107 @@ Manage Images in Datasets/Projects
 You can organize the data in the left-hand side tree by creating new Projects and Datasets. You can link the Images to the new or existing Datasets and Datasets to new or existing Projects.
 For HCS data, you can create new Screens and link Plates to these Screens.
 
-#. Use the Project |image2b| and Dataset |image2c| icons above the left-hand side tree or the right-click contextual menu to create new Datasets or Projects.
+#. Use the Project |image2b| and Dataset |image2c| icons above the left-hand
+   side tree or the right-click contextual menu to create new Datasets
+   or Projects.
 
-#. Drag and drop Images between Datasets and Datasets between Projects. For HCS data, drag and drop Plates between Screens.
+#. Drag and drop Images between Datasets and Datasets
+   between Projects. For HCS data, drag and drop Plates between Screens.
 
 #. Copy Images using the right-click context menu: 
-     - Select the Images to be copied, then right-click and click ``Edit > Copy Link``
-     - Select the Dataset you want to copy the Images to, right-click and click on ``Edit > Paste Link``
+     - Select the Images to be copied, then right-click
+       and click ``Edit > Copy Link``
+     - Select the Dataset you want to copy the Images to,
+       right-click and click on ``Edit > Paste Link``
 
        |image2d|
 
 .. warning::
-    **The ``Copy Link`` feature will only create new links between an Image and a Dataset, so that one Image becomes linked to multiple Datasets. This does not create a new independent copy of the Image.** The only way to create a fully independent copy in OMERO is to use the :ref:`Duplicate feature<Duplicatecli>`.
+    **The ``Copy Link`` feature will only create new links
```

Comment:
```
``Copy Link``
```
is still not formatted correctly
![image](https://user-images.githubusercontent.com/1644105/95476728-7302ec00-097f-11eb-9f66-ee151866241c.png)

Unrelated to this PR, but
![image](https://user-images.githubusercontent.com/1644105/95476794-89a94300-097f-11eb-983b-d5f061cc6abd.png)
should say be `Images` not Image` in `WARNING: One of more Image ...`

Reply:
The first issue mentioned here is fixed by https://github.com/ome/omero-guide-introduction/pull/17/commits/55dfa3d5f2bacb6742b368d59f0ca74b53ddeb90. The second issue must be fixed in the omero-web codebase I reckon, cc @will-moore

Reply:
The second issue mentioned here is fixed in https://github.com/ome/omero-guide-introduction/pull/17/commits/dc3f128b998d7d8b05446e608dbd4c66301f9972.

The state of thing is that the message in omero-web is gramatically wrong every time you select a single image. If youi select multiple images, the sentence sounds correct (some javascript is probably making a "clever" choice there based on number of images, but it would be better if it would not do anything of that sort, instead leaving the sentence in the form "images" at all times, cc @will-moore